### PR TITLE
Complete get heat rejection loops connected to baseline systems

### PR DESCRIPTION
**Pull request by weilixu**

### CODE REVIEW

**Review by yunjoonjung-PNNL — APPROVED** [review]
Great work! Thank you!

File: rct229/rulesets/ashrae9012019/ruleset_functions/get_heat_rejection_loops_connected_to_baseline_systems.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        A dictionary representing a ruleset model instance as defined by the ASHRAE229 schema
+
+    Returns -------
+    heat_reject_loop_list list
+        A list saves the ids of heat rejection loops in the model that serve baseline system types. If no qualifying
+        heat rejection loops, return an empty list.
+    """
+    primary_secondary_loops_dict = get_primary_secondary_loops_dict(rmi)
+    heat_rejection_loop_list = [
+        chiller["condensing_loop"]
```
Comment (charliepnnl) [inline rct229/rulesets/ashrae9012019/ruleset_functions/get_heat_rejection_loops_connected_to_baseline_systems.py:24]: I think this should be `chiller["condinsing_loop"]["id"]`.

**Review by charliepnnl — APPROVED** [review]
Just one comment that needs to be addressed.

File: rct229/rulesets/ashrae9012019/ruleset_functions/get_heat_rejection_loops_connected_to_baseline_systems.py (hunk 1 above)
Comment (weilixu) [reply rct229/rulesets/ashrae9012019/ruleset_functions/get_heat_rejection_loops_connected_to_baseline_systems.py:24]: The schema seems to be a reference to a [fluid loop](https://github.com/open229/ruleset-model-description-schema/blob/main/docs229/ASHRAE229.schema.json#L3356-L3360)

Comment (charliepnnl) [reply rct229/rulesets/ashrae9012019/ruleset_functions/get_heat_rejection_loops_connected_to_baseline_systems.py:24]: Yep. Your are correct.

**Pull request merged**